jest/private/jest_test.bzl

```diff
@@ -3,23 +3,35 @@
 load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
 load("@aspect_bazel_lib//lib:paths.bzl", "relative_file")
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
+load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _attrs = dicts.add(js_binary_lib.attrs, {
     "entry_point": attr.label(
         mandatory = True,
     ),
     "config": attr.label(
         doc = "jest config file, see TODO",
-        allow_single_file = [".js"],
+        allow_single_file = [".js", ".cjs", ".mjs", ".json"],
         mandatory = True,
     ),
+    "jest_repository": attr.string(
+        default = "jest",
+    ),
```

Comment:
I should add a doc string here.

Reply:
Looks like docstrings needed elsewhere as well in follow-ups; now that rules_ts is 1.0.0 this rule set is the next one to get up to par.